Mesh_3/examples/Mesh_3/mesh_3D_weighted_image.cpp

```diff
@@ -1,3 +1,5 @@
+#define CGAL_MESH_3_WEIGHTED_IMAGES_DEBUG
+
```

Comment:
I know the PR is a draft, but I want to check with you if you want to let that macro `CGAL_MESH_3_WEIGHTED_IMAGES_DEBUG` in the example code. Maybe that is intended. Or maybe you plan to remove it... let's leave this conversion unresolved until it is solved.

Reply:
I will remove it before making the PR "ready for review"